guides/common/modules/proc_creating-image-based-hosts-on-amazon-ec2-by-using-web-ui.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="creating-image-based-hosts-on-amazon-ec2-by-using-web-ui"]
+= Creating image-based hosts on Amazon EC2 by using {ProjectWebUI}
+
+[role="_abstract"]
+The Amazon EC2 provisioning process creates hosts from existing images on the Amazon EC2 server.
```

Comment:
```suggestion
You can perform image-based host provisioning on Amazon EC2 through {Project}.
```

Same would apply to CLI.
I dont' quite imagine that the whole AWS cloud is a single server.